canu.spec

```diff
@@ -40,17 +40,13 @@ Vendor: Cray HPE
 %setup -q
 
 %build
-# PIP and Setuptools updates.
 %{__python} -m pip install -U pyinstaller
 %{__python} -m pip install -q build
 %{__python} -m build
 %{__python} -m pip install dist/%{name}*.tar.gz
-git status
```

Comment:
were these only being used for debug?

Reply:
Yes. My bad, I left them in there in #184 (https://github.com/Cray-HPE/canu/pull/184/files#diff-264f24b02ad1146625c9cdb64f7a6fc2721dd9e662d7414a0293b5f4e8b8f65e).

Reply:
I was debugging why pyinstaller was failing due to an uncommitted file change, it ended up being from the metadata added to the spec file by the Prepare: RPMs step. I fixed it with [this](https://github.com/Cray-HPE/canu/blob/develop/Jenkinsfile.github#L58) but forgot to remove the debug lines in the `.spec`.